src/NServiceBus.RavenDB.AcceptanceTests/Config.cs

```diff
@@ -12,8 +12,7 @@ public void Configure(BusConfiguration config)
         documentStore = new DocumentStore
         {
             Url = "http://localhost:8083",
-            DefaultDatabase = Guid.NewGuid().ToString(),
-            ResourceManagerId = Guid.NewGuid() /* This is OK for ATT purposes */
```

Comment:
Didn't mean to commit that - result of my working on it in a separate branch where ResourceManagerId doesn't need to be set outright. I will fix.